mathics/builtin/files_io/files.py

```diff
@@ -296,7 +296,7 @@ class FilePrint(Builtin):
     }
 
     def apply(self, path, evaluation, options):
-        "FilePrint[path_ OptionsPattern[FilePrint]]"
+        "FilePrint[path_, OptionsPattern[FilePrint]]"
```

Comment:
Fixing OneIdentity, also come up this other bug...

Reply:
Another good catch!